src/lib/publicsuffixlist/wasm/publicsuffixlist.wat

```diff
@@ -39,13 +39,14 @@
 ;;  + u32: offset to array of children
 ;;  = 12 bytes
 ;;
-;;                                     // i32 /  i8
-;; const HOSTNAME_SLOT       = 0;      // jshint ignore:line
-;; const LABEL_INDICES_SLOT  = 256;    //  -- / 256
-;; const RULES_PTR_SLOT      = 100;    // 100 / 400
-;; const CHARDATA_PTR_SLOT   = 101;    // 101 / 404
-;; const EMPTY_STRING        = '';
-;; const SELFIE_MAGIC        = 2;
+;;                                      // i32 /  i8
```

Comment:
@gorhill all of this appears to be related to #3851.

Please also check that the `.wasm` file is identical now to the one in [publicsufffixlist.js](https://github.com/gorhill/publicsuffixlist.js).